<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCR文字识别</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openfile">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="PICTURE">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string>                                          识别图片放置处</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="opencamera">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头不检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>460</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>停止检测</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>351</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>检测该帧</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>baiduocr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>opencv+elm</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>识别方法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>logo放置处</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择logo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>识别图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>获取图片时间</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>百度OCR时间</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>是否自选矩形框</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>否</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>是</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>elm_in_elm时间</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>480</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>openfile</zorder>
   <zorder>opencamera</zorder>
   <zorder>textEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>stop</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>close</zorder>
   <zorder>comboBox</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>logo</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_2</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>label_4</zorder>
   <zorder>textEdit_4</zorder>
   <zorder>label_5</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>PICTURE</zorder>
   <zorder>label_6</zorder>
   <zorder>textEdit_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
